mathics/core/expression.py

```diff
@@ -1327,12 +1339,14 @@ def rules():
             new = Expression(head)
             new.elements = dirty_elements
 
-        # Step 8: Update the cache. Return the new compound Expression and indicate that no further evaluation is needed.
+        # Step 8: Update the cache. Return the new compound Expression and
+        #        indicate that no further evaluation is needed.
         new._timestamp_cache(evaluation)
         return new, False
 
     #  Now, let's see how much take each step for certain typical expressions:
-    #  (assuming that "F" and "a1", ... "a100" are undefined symbols, and n0->0, n1->1,..., n99->99)
+    #  (assuming that "F" and "a1", ... "a100" are undefined symbols, and
+    #  n0->0, n1->1,..., n99->99)
```

Comment:
The idea here was to add relative benchmarks to these cases. I thought I have done it, but they are not there... Maybe we can just move this as a pytest and directly benchmark it.

Reply:
Agreed - it would be better.